sdk/java/pulumi/src/main/java/com/pulumi/deployment/internal/TestOptions.java

```diff
@@ -15,48 +16,42 @@ public class TestOptions {
     private final boolean preview;
     private final List<ResourceTransformation> resourceTransformations;
 
-    public TestOptions() {
-        this(null, null, true);
-    }
-
-    public TestOptions(boolean preview) {
-        this(null, null, preview);
-    }
-
-    public TestOptions(@Nullable String projectName, @Nullable String stackName) {
-        this(projectName, stackName, true);
-    }
-
     /**
-     * @param projectName the test project name to use
-     * @param stackName the test stack name to use
-     * @param preview is the test a preview or a normal execution
+     * @param projectName             the test project name to use
+     * @param stackName               the test stack name to use
+     * @param preview                 is the test a preview or a normal execution
+     * @param resourceTransformations the test stack resource transformations
      */
-    public TestOptions(@Nullable String projectName, @Nullable String stackName, boolean preview) {
-        this.projectName = projectName != null ? projectName : "project";
-        this.stackName = stackName != null ? stackName : "stack";
+    public TestOptions(
```

Comment:
This change is good but breaking. I am guessing we don't have much code out there relying on TestOptions yet so this is ok just worth highlighting in the CHANGELOG that we have a new API here.